### PR TITLE
FOUR-10336: HOME: Add in the component "VIEW ALL"

### PR DESCRIPTION
**Pull request by CarliPinell**
## Solution
New filters were added to List Table new control
## Related Tickets & Packages
- FOUR-10336 package-screen-builder

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:deploy
ci:next
ci:processmaker:feature/FOUR-9486

### CODE REVIEW

File: src/components/renderer/form-new-request.vue [hunk 1]
```diff
@@ -82,7 +82,8 @@ export default {
             showControl: true,
             showAvatar: false,
             colorTextStart: "color: #57646F",
-            url: "#"
+            url: "#",
```
Comment (fagubla) [inline src/components/renderer/form-new-request.vue:85]: set URL as empty string

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-232f7e8907.eng.processmaker.net

File: src/components/renderer/form-list-table.vue [hunk 2]
```diff
@@ -66,19 +117,42 @@ export default {
...
+    this.fetch();
   },
   methods: {
     getData(data) {
       this.dataControl = data;
     },
     openExternalLink() {
       window.open(this.dataControl.url, "_blank");
+    },
+    fetch() {
```
[the remark this answers is not in the document]
Comment (CarliPinell) [reply src/components/renderer/form-list-table.vue:129]: Solved

File: src/components/renderer/form-new-request.vue (hunk 1 above)
Comment (CarliPinell) [reply src/components/renderer/form-new-request.vue:85]: solved

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-232f7e8907.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-232f7e8907.eng.processmaker.net

**Review by fagubla — APPROVED** [review]

**Pull request merged**